lib/internal/fs/utils.js

```diff
@@ -512,6 +516,30 @@ function warnOnNonPortableTemplate(template) {
   }
 }
 
+const defaultRmdirOptions = {
+  emfileWait: 1000,
+  maxBusyTries: 3,
+  recursive: false,
+};
+
+const validateRmdirOptions = hideStackFrames((options) => {
+  if (options === undefined)
+    return defaultRmdirOptions;
+  if (options === null || typeof options !== 'object')
+    throw new ERR_INVALID_ARG_TYPE('options', 'object', options);
+
+  options = { ...defaultRmdirOptions, ...options };
+
+  if (typeof options.recursive !== 'boolean')
+    throw new ERR_INVALID_ARG_TYPE('recursive', 'boolean', options.recursive);
+
+  validateInt32(options.emfileWait, 'emfileWait', 0);
```

Comment:
`validateUint32`?

Reply:
I went with `validateInt32()` because this gets passed to `setTimeout()`, which works with int32s. `validateUint32()` would be fine most of the time here, but would allow overflowing the timer.

Reply:
Ah, it's fine actually with that `min: 0` third argument which I missed on first glance.